commit 399ab68776c0075e10bef9ec85bd8736a2fedba6
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-09-13T15:28:44-04:00 (3 weeks ago)
    
    TOOLS-2303 sunset 14.2.0 image building support
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/README.md b/README.md
index 792eb89..ef14831 100644
--- a/README.md
+++ b/README.md
@@ -108,7 +108,6 @@ files. We will try to keep this table up to date:
 | triton-origin-x86\_64-18.4.0   | minimal-64-lts@18.4.0        | minimal-64-lts@18.4.0            | c2c31b00-1d60-11e9-9a77-ff9f06554b0f |
 | triton-origin-multiarch-18.1.0 | minimal-multiarch@18.1.0     | minimal-multiarch@18.1.0         | 1ad363ec-3b83-11e8-8521-2f68a4a34d5d |
 | triton-origin-multiarch-15.4.1 | minimal-multiarch-lts@15.4.1 | sdc-minimal-multiarch-lts@15.4.1 | 18b094b0-eb01-11e5-80c1-175dac7ddf02 |
-| -                              | -                            | sdc-base@14.2.0                  | de411e86-548d-11e4-a4b7-3bb60478632a |
 | -                              | -                            | sdc-smartos@1.6.3                | fd2cc906-8938-11e3-beab-4359c665ac99 |
 
 
